metrics_handler/job_status_handler_test.py

```diff
@@ -0,0 +1,116 @@
+# Lint as: python3
+"""Tests for job_status_handler."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import datetime
+import time
+import types
+
+from absl.testing import absltest
+import alert_handler
+import job_status_handler
+
+
+class FakeKubernetesStatus(object):
+  def __init__(self, active=None, succeeded=None, num_failures=None,
+               start_time=None, completion_time=None,
+               condition_transition_time=None, condition_reason=None):
+    self.active = active
+    self.succeeded = succeeded
+    self.failed = num_failures
+    self.start_time = start_time
+    self.completion_time = completion_time
+    if condition_transition_time:
+      condition = types.SimpleNamespace()  # Simple object with attrs.
+      condition.last_transition_time = condition_transition_time
+      condition.reason = condition_reason
+      self.conditions = [condition]
+    else:
+      self.conditions = None
+
+
+class JobStatusHandlerTest(absltest.TestCase):
+  def setUp(self):
+    self.logger = alert_handler.AlertHandler(
+      project_id=None,
+      write_to_logging=True,
+      write_to_error_reporting=False,
+      write_to_email=False)
+    self.handler = job_status_handler.JobStatusHandler(
+        "unused", "unused", "unused", self.logger)
+
+  def test_interpret_status_success_with_completion_time(self):
```

Comment:
Could these tests be done as parameterized tests? e.g. https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/metrics_handler/metrics_test.py#L142

Reply:
Switched to parameterized, PTAL @will-cromar